docs/src/advanced_features/concolic.md

```diff
@@ -92,18 +92,18 @@ In hybrid fuzzing, we combine this tracing + solving approach with more traditio
 The concolic tracing support in LibAFL is implemented using SymCC.
 SymCC is a compiler plugin for clang that can be used as a drop-in replacement for a normal C or C++ compiler.
 SymCC will instrument the compiled code with callbacks into a runtime that can be supplied by the user.
-These callbacks allow the runtime to construct a trace that similar to the previous example.
+These callbacks allow the runtime to construct a trace that is similar to the previous example.
 
 ### SymCC and its Runtimes
 
 SymCC ships with 2 runtimes: 
 
- * a 'simple' runtime that attempts to solve any branches it comes across using [Z3](https://github.com/Z3Prover/z3/wiki) and
- * a [QSym](https://github.com/sslab-gatech/qsym)-based runtime, which does a bit more filtering on the expressions and also solves using Z3.
+ * a 'simple' runtime that attempts to negate any branch conditions it comes across using [Z3](https://github.com/Z3Prover/z3/wiki) and
```

Comment:
Isn't solving the correct term here?

Reply:
Using solved for now, hope I'm understanding this correctly.

Reply:
solving is fine when it is explicitly stated what is being solved IMO. In this case the negation of the branch conditions encountered so far in order to reach yet unreached branches with the next input data.

Reply:
Proposed a wording in #986